modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/baseComponents/BaseAPIEndpointFields.tsx

```diff
@@ -180,82 +248,105 @@ export default function BaseAPIEndpointFields({
 
 	return (
 		<ClayForm>
-			{(editMode ?? false) && (
-				<ClayForm.Group className="disabled">
-					<label>
-						{Liferay.Language.get('method')}
-
-						<span className="ml-1 reference-mark text-warning">
-							<ClayIcon symbol="asterisk" />
-						</span>
-					</label>
-
-					<Select
-						disabled={true}
-						onClick={() => {}}
-						options={[{label: 'GET', value: 'get'}]}
-						required
-						searchable={false}
-						selectedOption={{label: 'GET', value: 'get'}}
-					/>
-				</ClayForm.Group>
-			)}
-
 			<ClayForm.Group
 				className={classNames({
-					'has-error': displayError.retrieveType,
+					'has-error': displayError.httpMethod,
 				})}
 			>
-				<label htmlFor="selectTrigger">
-					{Liferay.Language.get('retrieve-type')}
+				<label>
```

Comment:
question: why did you remove the htmlFor attribute? it makes sense for labels

Reply:
I'll add it!